script.js

```diff
@@ -0,0 +1,78 @@
+// ---------Responsive-navbar-active-animation-----------
+function test(){
```

Comment:
You can place this function inside document.ready and rename this function name and provide function description.